src/current/_includes/molt/replicator-flags.md

```diff
@@ -0,0 +1,85 @@
+| `--dlqTableName`             | `IDENT`    | The name of a table in the target schema for storing dead-letter entries.<br><br>**Default:** `replicator_dlq`                                |
+| `--flushPeriod`              | `DURATION` | Flush queued mutations after this duration.<br><br>**Default:** `1s`                                                                          |
+| `--flushSize`                | `INT`      | Ideal batch size to determine when to flush mutations.<br><br>**Default:** `1000`                                                             |
+| `--gracePeriod`              | `DURATION` | Allow background processes to exit.<br><br>**Default:** `30s`                                                                                 |
+| `--logDestination`           | `STRING`   | Write logs to a file. If not specified, write logs to `stdout`.                                                                               |
+| `--logFormat`                | `STRING`   | Choose log output format: `"fluent"`, `"text"`.<br><br>**Default:** `"text"`                                                                  |
+| `--metricsAddr`              | `STRING`   | A `host:port` on which to serve metrics and diagnostics.                                                                                      |
+| `--parallelism`              | `INT`      | The number of concurrent database transactions to use.<br><br>**Default:** `16`                                                               |
+| `--quiescentPeriod`          | `DURATION` | How often to retry deferred mutations.<br><br>**Default:** `10s`                                                                              |
+| `--retireOffset`             | `DURATION` | How long to delay removal of applied mutations.<br><br>**Default:** `24h0m0s`                                                                 |
+| `--scanSize`                 | `INT`      | The number of rows to retrieve from staging.<br><br>**Default:** `10000`                                                                      |
+| `--schemaRefresh`            | `DURATION` | How often a watcher will refresh its schema. If this value is zero or negative, refresh behavior will be disabled.<br><br>**Default:** `1m0s` |
+| `--sourceConn`               | `STRING`   | The source database's connection string.                                                                                                      |
+| `--stageMarkAppliedLimit`    | `INT`      | Limit the number of mutations to be marked applied in a single statement.<br><br>**Default:** `100000`                                        |
+| `--stageSanityCheckPeriod`   | `DURATION` | How often to validate staging table apply order (`-1` to disable).<br><br>**Default:** `10m0s`                                                |
```

Comment:
Again, I was not sure what "staging table" referred to, until I read down the list to `--stagingSchema`.